packages/bodiless-core/src/Types/ContextMenuTypes.tsx

```diff
@@ -36,6 +36,7 @@ export type TMenuOption = {
   group?: string;
   Component?: ComponentType<IContextMenuItemProps>;
   context?: string;
+  formTitle?: string;
```

Comment:
I don't think we should add this to TMenuOption, I think we should add it to [EditButtonOptions](https://github.com/wodenx/Bodiless-JS/blob/daacd0742f82923ef384f292281a79287c80a38b/packages/bodiless-core/src/Types/EditButtonTypes.tsx#L37), and retype [UseOverrides](https://github.com/wodenx/Bodiless-JS/blob/daacd0742f82923ef384f292281a79287c80a38b/packages/bodiless-components/src/Chameleon/types.tsx#L37) to match what we have in [asBodilessComponent](https://github.com/wodenx/Bodiless-JS/blob/daacd0742f82923ef384f292281a79287c80a38b/packages/bodiless-core/src/asBodilessComponent.tsx#L119).

In fact, I think it would be bst to export a `UseBodilessOverrides` type from `@bodiless/core` -- eventually, I think we will add support for this `formTitle` option (and probably for a `formDescription` and `formUnwrapText` to `withEditButton`.